include/pmacc/assert.hpp

```diff
@@ -55,4 +59,16 @@
  */
 #    define PMACC_ASSERT_MSG(expr, msg) (!!(expr)) ? ((void) 0) : pmacc::abortWithError(#    expr, __FILE__, __LINE__, msg)
 
+/** assert check
+ *
+ * if `NDEBUG` is not defined: macro expands to (void)0
```

Comment:
wrong: if `NDEBUG` is defined the macro expands to (void)0. // see line 28

Reply:
that's true, will change it for the old macros as well.